BuffTrigger.lua

```diff
@@ -1320,8 +1381,13 @@ function BuffTrigger.Modernize(data)
   end
 end
 
-function BuffTrigger.CanGroupShowWithZero(data)
-  local trigger = data.trigger;
+function BuffTrigger.CanGroupShowWithZero(data, triggernum)
```

Comment:
Since, now the dynamic information from all triggers can be shown, all the CanHave functions now get a triggernum and return the information of that trigger.